composer/algorithms/ema/README.md

```diff
@@ -73,14 +73,20 @@ EMA also uses a bit of extra compute to calculate the moving average. This can l
 
 ## Suggested Hyperparameters
 
-The Composer Trainer implementation of EMA has two hyperparameters:
+The Composer Trainer implementation of EMA has several hyperparameters:
 
 - `half_life` - The half life for terms in the average. A longer half life means old information is remembered longer, a shorter half life means old information is discared sooner.
-- `update_interval` - The period at which updates to the moving average are computed. A longer update interval means that updates are computed less frequently.
+- `update_interval` - The period at which updates to the moving average are computed. A longer update interval means that updates are computed less frequently. If left unspecified, this defaults to `1` in the units of `half_life`, or `1ba` if using `smoothing`.
+- `ema_start` -  The amount of training completed before SWA is applied. The default value is `'0.0dur'` which starts EMA at the start of training.
 
 A good typical starting value for `half_life` is `half_life="100ba"`, for a half life of 100 batches. At the same time, `update_interval` can be left unspecified which will default to `update_interval="1ba"`, or set to a larger value such as `update_interval="10ba"` to improve runtime. Shorter update intervals typically result in better generalization performance at the cost of somewhat increased runtime.
 
-Our implementation of EMA also provides the option to use the EMA weights as the training weights, which can be enabled by setting `train_with_ema_weights=True`. We reccomend leaving this off with the default value of `train_with_ema_weights=False.`
+For compatibility with other implementations, there is also an option to specify the value of `smoothing` directly.
+
+- `smoothing` - The coefficient representing the degree to which older observations are kept. The default (unspecified) value is `None`. Should only be used if `half_life` is not used.
```

Comment:
I do kind of want to keep it separate, because while it is an hparam, I feel pretty strongly that people should use `half_life` if they can. This is here mostly if someone really knows what they're doing and/or really wants to replicate a hyperparameter choice from another library.

Reply:
Sounds good 👍